ui/components/component-library/button-base/button-base.scss

```diff
@@ -0,0 +1,78 @@
+.mm-button {
+  position: relative;
+  height: 2.5rem;
```

Comment:
I've been suggesting we should be using pixel values outside of font-sizes just to reduce some cognitive load doing the calculations from design to dev. Let's discuss this. There is a great article by Josh Comeau on it regarding accessibility https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/